recipe/meta.yaml

```diff
@@ -6,35 +6,43 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/googlemaps-{{ version }}.tar.gz
-  sha256: 3055fcbb1aa262a9159b589b5e6af762b10e80634ae11c59495bd44867e47d88
+  url: https://github.com/googlemaps/google-maps-services-python/archive/refs/tags/v4.10.0.tar.gz
+  sha256: cc07f739731ba4e0a4586543ad14470695addff8f30840748b9c3b89e393aa36
 
 build:
-  noarch: python
+  skip: True  # [py<35]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
 
 requirements:
   host:
-    - python >=3.7
-    - requests >=2.20.0,<3.0
+    - python
     - pip
+    - setuptools
+    - wheel
+    - requests >=2.20.0,<3.0
   run:
-    - python >=3.7
+    - python
     - requests >=2.20.0,<3.0
 
 test:
   imports:
     - googlemaps
   commands:
     - pip check
+    - pytest
   requires:
     - pip
+    - pytest
+    - responses
+  source_files:
+    - tests
 
 about:
   home: https://github.com/googlemaps/google-maps-services-python
   summary: Python client library for Google Maps Platform
   license: Apache-2.0
+  license_family: APACHE
   license_file: LICENSE  
   description: |
     Use Python? Want to geocode something? Looking for directions? Maybe matrices of directions? This library brings the Google Maps API Web Services to your Python application. Analytics
```

Comment:
`doc_url` should be https://googlemaps.github.io/google-maps-services-python/docs/index.html

Reply:
Fixed it, thanks !